app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -1150,16 +1151,24 @@ public boolean onKeyDown(int keyCode, KeyEvent event) {
 
   @OnClick(R.id.tab_switcher_close_all_tabs)
   void closeAllTabs() {
+    rotate(closeAllTabsButton);
     webViewList.clear();
     tabsAdapter.notifyDataSetChanged();
+    openHomeScreen();
+    updateTabSwitcherIcon();
+  }
+
+  //opens home screen when user closes all tabs
+  private void openHomeScreen() {
     new Handler().postDelayed(() -> {
       if (webViewList.size() == 0) {
         newTab(HOME_URL);
+        hideTabSwitcher();
       }
-    }, 1500);
-    updateTabSwitcherIcon();
+    }, 300);
   }
 
+
```

Comment:
Remove this line.

Reply:
@abdulwd Done. Thanks for reviewing.

Reply:
@abdulwd I have removed the line. Could you please review?